app/javascript/orangelight/availability.es6

```diff
@@ -427,16 +421,17 @@ export default class AvailabilityUpdater {
 
     return link;
   };
-
-  due_date(date_string) {
-    if (date_string == null) { return ""; }
-    return ` - ${date_string}`;
-  };
-
-  on_site_use_marquand_location(location) {
-    let marquand_location = ["marquand$stacks", "marquand$pj", "marquand$ref","marquand$ph", "marquand$fesrf"]
-    return marquand_location.includes(location);
-  };
+  
+  // Set status for specific Marquand locations and location RES_SHARE$IN_RS_REQ
+  checkSpecialLocation(location, availability_element) {
```

Comment:
check uncaught TypeError: Cannot read properties of null (reading 'text'),  99118399983506421 @christinach